board/opentrons/ot2/rootfs-overlay/usr/bin/ot-status-leds

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# Monitor system status and update LEDs accordingly
+# D2 ) Ethernet LED  : STATUS_LED_1  => Pin15    => bcm22
+# D3 ) Wifi LED      : STATUS_LED_2  => Pin27    => bcm00
+# D4 ) Heartbeat LED : STATUS_LED_3  => Pin22    => bcm25
+# D5 ) UNDEFINED_LED : STATUS_LED_4  => Pin28    => bcm01
+
+import gpiod
+import enum
+import asyncio
+from opentrons.system import nmcli
+
+
+CHIP = gpiod.Chip('gpiochip0')
+
+ETH_LED = 22
+WIFI_LED = 0
+HEARTBEAT_LED = 25
+UNDEFINED_LED = 1
+
+
+class Line(enum.Enum):
+    ETH = enum.auto()
+    WIFI = enum.auto()
+    HEARTBEAT = enum.auto()
+    UNDEFINED = enum.auto()
+
+
+GPIO_LINES = {
+    Line.ETH: CHIP.get_line(ETH_LED),
+    Line.WIFI: CHIP.get_line(WIFI_LED),
+    Line.HEARTBEAT: CHIP.get_line(HEARTBEAT_LED),
+    Line.UNDEFINED: CHIP.get_line(UNDEFINED_LED)
```

Comment:
I feel like these 3 structures could be combined. For example, instead of doing

```python
class Line(enum.Enum):
    ETH = enum.auto()
    # ...
```
You could do:

```python
class Line(enum.Enum):
    ETH = 22
    # ...
```

(And you *might* even be able to do:)

```python
class Line(enum.Enum):
    ETH = CHIP.get_line(22)
    # ...
```

(But I would check the contracts of `enum.Enum` and `CHIP.get_line()` to make sure that that wouldn't break any assumptions of static-ness.)

Reply:
Not sure that this is any better than @SyntaxColoring approach, but another approach could be... in psuedo code:

```
class Line(int, Enum):
    ETH = 22
    WIFI = 0
    .....
    def __init__(self, pin):
        # get the line in the Enum constructor
        self._line = CHIP.get_line(pin)
    
    @property
    def line(self):
        return self._line
```
 
And to access:

```
# Pin
Line.ETH
# Line
Line.ETH.line